ONSdigital/sdx-transform-cs
# Construction

## What? and Why?
Needed to add the construction survey to the transformer. 

## How to test
- Open console
- Submit a 228.0001.json
- Check the output in the FTP.  The transformation should match what's on the screenshot (attached to the trello card) and the images should look reasonable

## Checklist
